### DISCUSSION TITLE
vcpkg install fails on extracting 7zip

### DISCUSSION THREAD

**Discussion by mwilson75** [microsoft/vcpkg#20533] [Q&A]
OS: windows 10

I followed the directions on the  Quick Start: Windows section and cannot seem to install anything using vcpkg.

I get the error:

.\vcpkg\vcpkg install eigen3:x64-windows
Computing installation plan...
The following packages will be built and installed:
    eigen3[core]:x64-windows -> 3.3.9#1
Detecting compiler hash for triplet x64-windows...
A suitable version of git was not found (required v2.32.0). Downloading portable git v2.32.0...
Extracting git...
A suitable version of 7zip was not found (required v19.0.0). Downloading portable 7zip v19.0.0...
Extracting 7zip...
msiexec failed while extracting 'C:\src\vcpkg\vcpkg\downloads\7z1900-x64.msi' with message:
The system cannot find the path specified.

What I am confused about is that path does exist. I do not understand what is wrong.

**Comment by dg0yt** [comment]
Did you simply retry? Not that is just Anti-Virus locking the file for a moment.

**Comment by DimRochette** [comment]
You may try to run it one time with elevated privilege, just for 7zip to be installed. 

There is no other workaround to my knowledge.

It breaks on our jenkins since the move to msi =>issue https://github.com/microsoft/vcpkg/issues/19490
